frontend/packages/dev-console/src/components/edit-application/__tests__/edit-application-utils.spec.ts

```diff
@@ -108,6 +117,14 @@ describe('Edit Application Utils', () => {
     ).toEqual(gitImportInitialValuesWithHealthChecksEnabled);
   });
 
+  it('should get correct data for fileupload values', () => {
+    const { buildConfig, editAppResource } = appResources;
+    expect(getFileUploadValues(editAppResource.data, buildConfig.data)).toEqual({
+      fileUpload: { name: 'demo-app.jar', value: '', javaArgs: undefined },
+      runtimeIcon: 'python',
+    });
```

Comment:
will it make sense to update `buildConfig` mock data to have `java` as runtime as Upload Jar will always have that

Reply:
This is just mock data. I don't see any difference using java or python here.

Reply:
This is just mock data. I don't see any difference using java or python here.